$NetBSD: patch-src_CMakeLists.txt,v 1.11 2021/12/20 20:59:42 nia Exp $

Werror with an arbitrary toolchain considered harmful.

Don't link with the C compiler, CMake's CXXABI detection may insert
C++ libraries into LIBS.

--- src/CMakeLists.txt.orig	2021-09-10 06:20:12.000000000 +0000
+++ src/CMakeLists.txt
@@ -739,10 +739,6 @@ find_package(Pugixml 1.2 REQUIRED)
 include_directories(SYSTEM ${Pugixml_INCLUDE_DIRS})
 list(APPEND LIBS ${Pugixml_LIBRARIES})
 
-if(NOT SOURCE_PACKAGE)
-  add_definitions(-Werror -Wfatal-errors )
-endif()
-
 if(CMAKE_CXX_COMPILER_ID STREQUAL "AppleClang" OR CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
   # do not error out on "loop not vectorized" clang OpenMP warnings
   add_definitions("-Wno-error=pass-failed")
@@ -907,7 +903,7 @@ if(APPLE)
   set_target_properties(lib_darktable PROPERTIES MACOSX_RPATH TRUE)
 endif(APPLE)
 set_target_properties(lib_darktable PROPERTIES OUTPUT_NAME darktable)
-set_target_properties(lib_darktable PROPERTIES LINKER_LANGUAGE C)
+#set_target_properties(lib_darktable PROPERTIES LINKER_LANGUAGE C)
 
 #
 # Link external libs
